modules/ROOT/pages/release-notes.adoc

```diff
@@ -26,26 +26,7 @@ endif::[]
 
 == Internal Beta
 
-*April 22, 2019*
-
-=== New Features
-
-* one
-* two
-
-=== Fixed in this Release
-
-* one
-* two
-
-=== Known Issues
-
-* one
-* two
-
-== Internal Alpha
-
-*April 8, 2019*
+*Month 2, 2019*
```

Comment:
Month? Is that just a place holder?

Reply:
Yes, those are placeholders, and they are commented out and not visible.